src/persist-client/src/operators/shard_source.rs

```diff
@@ -66,27 +67,28 @@ use crate::{PersistLocation, ShardId};
 /// using [`timely::dataflow::operators::generic::operator::empty`].
 ///
 /// [advanced by]: differential_dataflow::lattice::Lattice::advance_by
-pub fn shard_source<K, V, D, F, G>(
+pub fn shard_source<T, K, V, D, F, G>(
```

Comment:
nit: make this `<K, V, T, D,...` to match the rest of persist

Reply:
ah didnt realize